{"uuid":"00f9040d-41f3-4923-ac3b-3bb1c07a27a0","historyId":"32d4d9840db85ff4dc18f210c06ddadd","fullName":"test.LoginTest.testValidLogin","labels":[{"name":"package","value":"test.LoginTest"},{"name":"testClass","value":"test.LoginTest"},{"name":"testMethod","value":"testValidLogin"},{"name":"parentSuite","value":"Smoke Test Suite"},{"name":"suite","value":"Smoke Test"},{"name":"subSuite","value":"test.LoginTest"},{"name":"host","value":"DESKTOP-JBP6348"},{"name":"thread","value":"10136@DESKTOP-JBP6348.TestNG-test=Smoke Test-1(30)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"story","value":"As a user I should be able to login to the application"},{"name":"epic","value":"User Account"},{"name":"feature","value":"Login"},{"name":"severity","value":"blocker"}],"links":[],"name":"Verify that a valid user can login to the application","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [elka] but found [Телефон:]","trace":"java.lang.AssertionError: expected [elka] but found [Телефон:]\r\n\tat org.testng.Assert.fail(Assert.java:96)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:776)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:137)\r\n\tat org.testng.Assert.assertEquals(Assert.java:118)\r\n\tat org.testng.Assert.assertEquals(Assert.java:453)\r\n\tat org.testng.Assert.assertEquals(Assert.java:463)\r\n\tat test.LoginTest.testValidLogin(LoginTest.java:39)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)\r\n\tat org.testng.internal.Invoker.invokeMethod(Invoker.java:583)\r\n\tat org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)\r\n\tat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\r\n\tat java.base/java.lang.Thread.run(Thread.java:1583)\r\n"},"stage":"finished","description":"Verify that a valid user can login to the application","steps":[],"attachments":[],"parameters":[],"start":1712579589116,"stop":1712579614149}